src/main/java/com/desiato/puresynth/models/AuthenticatedUser.java

```diff
@@ -0,0 +1,19 @@
+package com.desiato.puresynth.models;
+
+public class AuthenticatedUser {
```

Comment:
Why not record?

Reply:
I tried that, but then I cannot use the getters

Reply:
ok! I'll do that. Thanks